releng/org.eclipse.cdt.target/cdt.target

```diff
@@ -14,6 +14,9 @@
 			<unit id="org.eclipse.test.feature.group" version="0.0.0" />
 			<unit id="org.eclipse.unittest.ui" version="0.0.0" />
 		</location>
+		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+			<repository location="https://download.eclipse.org/mylyn/updates/release/latest/" />
```

Comment:
@MarkZ3 SimRel version of Mylyn WikiText
Since LSP4E is full of innovations I wonder why they still don't upgrade Mylyn WikiText.

Reply:
That's an easy answer - people doing and caring for releng are so few and spread over so many projects that such things are easily missed. Best would be if PRs are provided for such changes wherever needed.
I have to admit that with lsp4e being with very few deps I manage to find time for updating its deps only few times per year (and usually this is driven by LSP4J ).

Reply:
> @MarkZ3 SimRel version of Mylyn WikiText

Sorry is this a question or suggestion?